src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figure = new FigureSupplier();
+        int[] array = new int[6];
+        for (int i = 0; i < array.length; i++) {
+            if (i < array.length / 2) {
```

Comment:
All magic numbers in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names